syntax = "proto3";

package pb;

message UserRequest {
    string id = 1;
    string username = 2;
    string image = 3;
}

message UserResponse {}

message IDRequest {
    string id = 1;
}

service Tasks {
    rpc AddUser (UserRequest) returns (UserResponse);
    rpc ChangeUser (UserRequest) returns (UserResponse);
    rpc DeleteUser(IDRequest) returns (UserResponse);
}